packages/components/src/components/progressbar/progressbar.e2e-test.ts

```diff
@@ -0,0 +1,143 @@
+import { expect } from '@playwright/test';
+import { ComponentsPage, test } from '../../../config/playwright/setup';
+import StickerSheet from '../../../config/playwright/setup/utils/Stickersheet';
+import { VARIANT } from './progressbar.constants';
+
+type SetupOptions = {
+  componentsPage: ComponentsPage;
+  value?: string;
+  variant?: string;
+  label?: string;
+  error?: boolean;
+  'help-text'?: string;
+  'data-aria-label'?: string;
+};
+
+const setup = async (args: SetupOptions) => {
+  const { componentsPage, ...restArgs } = args;
+  await componentsPage.mount({
+    html: `
+      <mdc-progressbar
+        ${restArgs.value ? `value="${restArgs.value}"` : ''}
+        ${restArgs.variant ? `variant="${restArgs.variant}"` : ''}
+        ${restArgs.label ? `label="${restArgs.label}"` : ''}
+        ${restArgs.error ? 'error' : ''}
+        ${restArgs['help-text'] ? `help-text="${restArgs['help-text']}"` : ''}
+        ${restArgs['data-aria-label'] ? `data-aria-label="${restArgs['data-aria-label']}"` : ''}
+      >
+      </mdc-progressbar>
+    `,
+    clearDocument: true,
+  });
+  const progressbar = componentsPage.page.locator('mdc-progressbar');
+  await progressbar.waitFor();
+  return progressbar;
+};
+
+const attributeTestCases = async (componentsPage: ComponentsPage) => {
+  const progressbar = await setup({ componentsPage });
+  await test.step('default attributes', async () => {
+    await expect(progressbar).toHaveAttribute('variant', 'default');
+    await expect(progressbar).toHaveAttribute('value', '0');
+  });
+  await test.step('attributes are set properly', async () => {
+    await progressbar.evaluate((el) => el.setAttribute('value', '30'));
+    await expect(progressbar).toHaveAttribute('value', '30');
+  });
+};
+
+const runVisualRegressionTest = async (componentsPage: ComponentsPage, progressbarStickerSheet: any) => {
+  // eslint-disable-next-line max-len
+  progressbarStickerSheet.setAttributes({ label: 'Label', 'help-text': 'Helper text', 'data-aria-label': 'This is a progress bar' });
```

Comment:
```suggestion
  progressbarStickerSheet.setAttributes({ 
    label: 'Label',
   'help-text': 'Helper text', 
   'data-aria-label': 'This is a progress bar' 
   });
```

Let's try not to add eslint-disable for max-len. Instead, you could split the code into multiple lines.